uint16 protocol
uint16 pkgId
uint32[6] spare
uint32 dataSize